### PR TITLE
feat(homepage): kenya map.

### PR DESCRIPTION
**Pull request by NofarGizra**
#34

### CODE REVIEW

**Comment by NofarGizra** [comment]
Update:
Map is able to automatically focus on Kenya.
Now working on coloring areas (in progress).

**Comment by NofarGizra** [comment]
Update:
Kenya is now displayed with all the different regions, but they are all the same color.

**Comment by NofarGizra** [comment]
iblimapp

**Comment by NofarGizra** [comment]
Update:
We now have colors on the map, including a legend!
We also support the 2 different indexes - one for LRLD and one for SRSD. The will switch automatically according to the current date.

**Comment by NofarGizra** [comment]
LRLD

map1

**Comment by NofarGizra** [comment]
SRSD

map2

**Comment by niryariv** [comment]
awesome!

**Comment by NofarGizra** [comment]
Update:
Added a custom block under the legend for displaying details.
When hovering a division, the division changes color to white, and its name and percents are displayed in the new custom block.

**Comment by NofarGizra** [comment]
map3

File: apps/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[![Build Status](https://magnum.travis-ci.com/Gizra/ethosia-client.svg?token=p2M1EeCrd3dY32WxWj3X&branch=master)](https://magnum.travis-ci.com/Gizra/ethosia-client)
+
+Ethosia Client
```
Comment (amitaibu) [inline apps/README.md]: Remove this name here and other places

File: apps/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[![Build Status](https://magnum.travis-ci.com/Gizra/ethosia-client.svg?token=p2M1EeCrd3dY32WxWj3X&branch=master)](https://magnum.travis-ci.com/Gizra/ethosia-client)
```
Comment (ceoaliongroo) [inline apps/README.md]: Change to IBLI travis

File: apps/app/css/main.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (ceoaliongroo) [inline apps/app/css/main.css]: remove this file.

File: apps/app/data-mock/geo.get.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (ceoaliongroo) [inline apps/app/data-mock/geo.get.json]: remove file.

File: apps/app/index.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+    <!-- build:css styles/vendor.css -->
+    <!-- bower:css -->
+    <link rel="stylesheet" href="bower_components/leaflet-dist/leaflet.css" />
+    <!-- endbower -->
+    <!-- endbuild -->
+    <!-- build:css({.tmp,app}) styles/main.css -->
+    <link rel="stylesheet" href="styles/main.css">
+    <!-- endbuild -->
+    <link rel="stylesheet" type="text/css" href="http://cloud.github.com/downloads/lafeber/world-flags-sprite/flags32.css" />
```
Comment (ceoaliongroo) [inline apps/app/index.html]: remove it.

File: apps/app/index.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+    <script src="bower_components/angular-apimock/dist/angular-apimock.min.js"></script>
+
+    <!-- endbuild -->
+
+        <!-- build:js({.tmp,app}) scripts/scripts.js -->
+        <script src="scripts/app.js"></script>
+        <script src="scripts/controllers/main.js"></script>
+        <script src="scripts/services/boxes.js"></script>
+        <script src="scripts/services/ethosiahttpinterceptor.js"></script>
```
Comment (ceoaliongroo) [inline apps/app/index.html]: - Change name of the file to: ibliHttpInterceptor.js
- Remove boxes.js

File: apps/app/scripts/controllers/main.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+      }
+    });
+
+    // Define a function to be executed when hovering a division.
+    $scope.$on("leafletDirectiveMap.geojsonMouseover", function(ev, leafletEvent) {
+      divisionMouseover(leafletEvent);
+    });
+
+    // Get the CSV of indexes, according to the current season.
+    $http.get('csv/indexes' + currentSeason + '.csv').success(function(data) {
```
Comment (ceoaliongroo) [inline apps/app/scripts/controllers/main.js]: Move to a service.

File: apps/app/scripts/services/ethosiahttpinterceptor.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+angular.module('ibliApp')
+  .factory('ethosiaHttpInterceptor', function ($q, BACKEND_URL) {
```
Comment (ceoaliongroo) [inline apps/app/scripts/services/ethosiahttpinterceptor.js]: replace ethosia => ibli

File: apps/package.json [hunk 9]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ethosiaclient",
```
Comment (ceoaliongroo) [inline apps/package.json]: replace ethosia ==> ibli

**Comment by NofarGizra** [comment]
Update:
- Technical stuff: Moved all data handling from the controller to a service.
- Added "maxBounds" - you now cannot have Kenya outside the map, it is always in the middle (even if you zoom in or zoom out).

File: apps/app/scripts/controllers/main.js [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+    });
+
+    // Custom control for displaying name of division and percent on hover.
+    $scope.controls = {
+      custom: []
+    };
+    var hoverInfoControl = L.control();
+    hoverInfoControl.setPosition('bottomleft');
+    hoverInfoControl.onAdd = function () {
+      return $compile(angular.element('<hover-info></hover-info>'))($scope)[0];;
```
Comment (ceoaliongroo) [inline apps/app/scripts/controllers/main.js]: remove ;

File: apps/app/scripts/controllers/main.js [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    }
```
Comment (ceoaliongroo) [inline apps/app/scripts/controllers/main.js]: add ;

File: apps/app/styles/main.scss [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+// endbower
+
+.browsehappy {
+    margin: 0.2em 0;
+    background: #ccc;
+    color: #000;
+    padding: 0.2em 0;
+}
+
+/* Space out content a bit */
```
Comment (ceoaliongroo) [inline apps/app/styles/main.scss]: Remove until line 97.

File: apps/app/scripts/directives/hover-info.js [hunk 13]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('ibliApp')
+  .directive('hoverInfo', function ($compile) {
```
Comment (ceoaliongroo) [inline apps/app/scripts/directives/hover-info.js]: Remove $compile, it's not needed.

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**